### DISCUSSION TITLE
Attempted to call an undefined method named "setMiddlewares" of class "Doctrine\DBAL\Configuration"

### DISCUSSION THREAD

**Discussion by Stevenzors** [doctrine/DoctrineBundle#1498] [Q&A] [answered]
Hi, 

I updated on doctrine-bundle 2.6.1 and still got the same error. 

Do you have an idea ?

Thanks

**Comment by dmaicher** [comment, marked as answer]
Likely you are using Sentry? then you need to wait for [this](https://github.com/getsentry/sentry-symfony/pull/608) to be released

**Reply by Stevenzors** [reply]
Thanks :)
